### ISSUE TITLE
Screen brightness setting is not saved after warm boot.

### ISSUE THREAD

**Issue by johanes2115** [Dasharo/dasharo-issues#444]
**Dasharo version**
Dasharo Enterprise v0.9.0

**Dasharo variant**
nv4x_adl, nv4x_tgl

**Affected component(s) or functionality**
Screen brightness

**Brief summary**
Screen brightness setting is not saved after warm boot.

**How reproducible**
100%

**How to reproduce**

1. Power on the DUT.
2. Log into the OS.
3. Set desired screen brightness level.
4. Warm boot the DUT.

**Expected behavior**
Screen brightness level should remain the same.

**Actual behavior**
Sreen brightness levels resets to max.

**Screenshots**

**Additional context**

**Solutions you've tried**

**Comment by miczyg1** [comment]
Screen brightness was never persistent AFAIK. Or was it?

**Comment by johanes2115** [comment]
Seemed like a bug to me, if it is how the platforms should behave I can remove the issue.

**Comment by miczyg1** [comment]
Let me rephrase it:

Did we ever test screen brightness persistence? 

@sulewskiprzemyslaw cc
